src/org/ods/Context.groovy

```diff
@@ -131,4 +137,35 @@ interface Context {
     boolean getCiSkip()
 
     def setEnvironmentCreated(boolean created)
+
+    // Whether CI skip is enabled
+    boolean getCiSkipEnabled()
+
+    // Enable/disable CI skip is enabled
+    def setCiSkipEnabled(boolean ciSkipEnabled)
+
+    // Whether Bitbucket notification is enabled
+    boolean getBitbucketNotificationEnabled()
+
+    // nable/disable Bitbucket notification
+    def setBitbucketNotificationEnabled(boolean bitbucketNotificationEnabled)
+
+    // Whether local checkout is enabled
+    boolean getLocalCheckoutEnabled()
+
+    // Enable/disable local checkout
+    def setLocalCheckoutEnabled(boolean localCheckoutEnabled)
+
+    // Whether display name update is enabled
+    boolean getDisplayNameUpdateEnabled()
+
+    // Enable/disable display name update
+    def setDisplayNameUpdateEnabled(boolean displayNameUpdateEnabled)
+
+    // Whether mail notification is enabled
+    boolean getMailNotificationEnabled()
+
+    // Enable/disable mail notification
+    def setMailNotificationEnabled(boolean mailNotificationEnabled)
```

Comment:
It seems like we are not actually use all the `set*` methods - maybe better to remove them then?

Reply:
Removed all unused methods.